package.json

```diff
@@ -109,6 +109,7 @@
     "redux-mock-store": "0.0.3",
     "selenium-standalone": "~4.9.0",
     "sinon": "2.0.0-pre",
+    "teaspoon": "~6.1.2",
```

Comment:
I know we've used `teaspoon` in the past, but isn't `enzyme` (https://github.com/airbnb/enzyme) the new standard? Ping @jeromemacias @brikou @RobinBressan @djhi